Tests/Marketplace/configure_and_install_packs.py

```diff
@@ -39,16 +39,15 @@ def install_packs_from_content_packs_to_install_path(servers, pack_ids, marketpl
     Install pack_ids from "$ARTIFACTS_FOLDER/content_packs_to_install.txt" file, and packs dependencies.
 
     Args:
-        hostname:
         pack_ids: the pack IDs to install.
         servers: XSIAM or XSOAR Servers to install packs on it.
     """
-    install_packs_one_by_one: bool = marketplace_tag_name == XSIAM_MP
+    use_multithreading = marketplace_tag_name != XSIAM_MP
```

Comment:
why you changed from `==` to `!=`?

Reply:
I changed the`install_packs_one_by_one` flag to `multithreading` on https://github.com/demisto/content/pull/27405/commits/57b3e7d37ab17bf35f5b491fb0545218d039d0ab since it was confusing (at least to me), but now it's the opposite (`install_packs_one_by_one` is like  `not multithreading`), so I had to reverse it.

To make it easier to follow - this condition means that we install packs one by one (without multithreading) when the `XSIAM_MP` is used. So if the tag is different, we use multithreading.